about.css

```diff
@@ -0,0 +1,859 @@
+  height: 2.8rem;
+  width: 7.5rem;
+}
+
+/* \\\\\\\\Footer Section//////// */
+.footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  height: 4rem;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.footer-logo {
+  margin-top: -4.5rem;
+}
+
+.footer-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.footer-container img {
+  margin-left: 1rem;
+  width: 9rem;
+  height: 3rem;
+}
+
+.f-c-h {
+  display: flex;
+  text-align: start;
+  color: #272a31;
+  width: 9rem;
+  font-size: 14px;
+  margin-top: -2rem;
+}
+
+/* Mobile Version */
+@media only screen and (min-width: 768px) {
+  .top-menu {
+    display: flex;
+    justify-content: flex-end;
+    background-color: #272a31;
+  }
+
+  .nonee {
+    display: none;
+  }
+
+  .none {
+    display: block;
+  }
+
+  .top-menu ul {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    gap: 10px;
+    padding: 10px 570px 10px 0;
+  }
+
+  .navbar img,
+  .navbar .nav-button,
+  .page-nav {
+    display: block;
+  }
+
+  header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .checkbox,
+  .hamburger-lines {
+    display: none !important;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #fff;
+    box-shadow: 5px 10px 8px #d4d4d4;
+    background-image: none;
+  }
+
+  .navbar img {
+    margin-left: 1rem;
+    margin-top: -0.8rem;
+  }
+
+  .navbar .nav-items {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: row;
+    list-style: none;
+    padding: 0;
+    height: 0;
+    width: 0;
+    background-color: #fff;
+    opacity: 1;
+    transform: none;
+    margin-top: 0.9rem;
+  }
+
+  .navbar ul a {
+    text-decoration: none;
+    color: #5c5a5a;
+    margin: 0;
+    font-size: 16px;
+  }
+
+  .page-nav ul a {
+    text-decoration: none;
+    color: #5c5a5a;
+  }
+
+  .navbar ul li a:hover {
+    color: #ec5242;
+    font-size: 18px;
+    animation: 0.5s ease;
+  }
+
+  .navbar .nav-button {
+    border-color: #ec5242;
+    border-width: 7px;
+    color: #ec5242;
+    background-color: #fff;
+    font-size: 16px;
+    padding: 12px;
+    margin: -0.5rem 15rem 0 1rem;
+  }
+
+  .navbar .nav-button:hover {
+    color: #fff;
+    background-color: #ec5242;
+    font-size: 18px;
+  }
+
+  .page-nav ul {
+    display: flex;
+    list-style: none;
+    gap: 0.3rem;
+  }
+
+  /* About Content */
+  .about-content .content-title {
+    color: #ec5242;
+    font-size: 40px;
+    font-weight: 400;
+  }
+
+  .about-header-content {
+    background-image: url(img/desktop-bg.png);
+    background-size: cover;
+    padding: 3rem 0 0;
+    margin: 0 0 0 -1rem;
+    height: 100%;
+  }
```

Comment:
- [ ] As mentioned by the previous code reviewer,  your design is not matching the given requirement, please align the content of your `about-header-content` class properties and its contents according to the design. 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student about home](https://github.com/Edgahkipkemoi/capstone/assets/43528594/40609a6a-0eac-4e80-8a53-3bd3e0ce5f7f)

</td>
      <td>

![Design about home](https://github.com/Edgahkipkemoi/capstone/assets/43528594/5a5fcac2-9115-407b-a04b-20c80b6d9b9f)

</td>
    </tr>
  </table>